cfn-resources/private-endpoint-regional-mode/cmd/resource/resource.go

```diff
@@ -177,7 +165,7 @@ func isRegModeSettingExists(currentModel *Model, client *mongodbatlas.Client) bo
 	return isExists
 }
 
-func regionalPrivateEndpointToModel(currentModel Model, regPrivateMode *mongodbatlas.RegionalizedPrivateEndpointSetting) *Model {
+func regionalPrivateEndpointToModel(currentModel Model) *Model {
```

Comment:
```suggestion
func responseModel(currentModel Model) *Model {
```
was confused with current name, what do you think?

Reply:
updated